openedx/features/course_experience/views/course_message.py

```diff
@@ -0,0 +1,130 @@
+"""
```

Comment:
Nit: I'd rename this file as ``course_messages.py``, or maybe even ``course_home_messages.py``.